packages/workout-spa-editor/src/components/pages/CalendarHeader.test.tsx

```
@@ -0,0 +1,130 @@
+    pending: null,
+    confirmStart: vi.fn(),
+    cancelRequest: vi.fn(),
+  },
+  hasAnyWorkouts: true,
+  hasWeekWorkouts: true,
+  hasReadyWorkouts: false,
+  hasAiProvider: true,
+  extensionInstalled: true,
+  latestWorkout: null,
+  handleGoToLatest: vi.fn(),
+  // biome-ignore — partial type for test
```

Biome (2.4.11)

[error] Failed to parse category "—". Example of suppression: // biome-ignore lint: reason

(suppressions/parse)